<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="395" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="395" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:32Z">
      <class name="testcases.TestCase1">
        <test-method status="PASS" signature="setReport()[pri:0, instance:testcases.TestCase1@147ed70f]" name="setReport" is-config="true" duration-ms="104" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setReport -->
        <test-method status="PASS" signature="doLogin()[pri:0, instance:testcases.TestCase1@147ed70f]" name="doLogin" duration-ms="9" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.TestCase1@147ed70f]" name="tearDown" is-config="true" duration-ms="16" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=doLogin status=SUCCESS method=TestCase1.doLogin()[pri:0, instance:testcases.TestCase1@147ed70f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="doUserReg()[pri:0, instance:testcases.TestCase1@147ed70f]" name="doUserReg" duration-ms="2" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Executing registration test]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Executing registration test
at org.testng.Assert.fail(Assert.java:96)
at testcases.TestCase1.doUserReg(TestCase1.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.TestCase1@147ed70f]" name="tearDown" is-config="true" duration-ms="2" started-at="2021-11-01T08:56:31Z" finished-at="2021-11-01T08:56:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=doUserReg status=FAILURE method=TestCase1.doUserReg()[pri:0, instance:testcases.TestCase1@147ed70f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="isSkip()[pri:0, instance:testcases.TestCase1@147ed70f]" name="isSkip" duration-ms="1" started-at="2021-11-01T08:56:32Z" finished-at="2021-11-01T08:56:32Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping test]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping test
at testcases.TestCase1.isSkip(TestCase1.java:79)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.TestCase1@147ed70f]" name="tearDown" is-config="true" duration-ms="0" started-at="2021-11-01T08:56:32Z" finished-at="2021-11-01T08:56:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=isSkip status=SKIP method=TestCase1.isSkip()[pri:0, instance:testcases.TestCase1@147ed70f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:testcases.TestCase1@147ed70f]" name="endReport" is-config="true" duration-ms="239" started-at="2021-11-01T08:56:32Z" finished-at="2021-11-01T08:56:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- testcases.TestCase1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
